src/Engine/Manager/RenderManager.cpp

```diff
@@ -55,6 +56,9 @@ RenderManager::RenderManager() {
     if (Managers().vrManager->Active())
         hmdRenderSurface = new Video::RenderSurface(Managers().vrManager->GetRecommendedRenderTargetSize());
 
+    //Init shadowpass.
+    shadowPass = new Video::ShadowPass();
```

Comment:
Memory leak. Shadow pass is never deleted.

Reply:
Fixed